tests/subsys/bluetooth/fast_pair/crypto/Kconfig

```diff
@@ -4,6 +4,31 @@
 # SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
 #
 
+config FP_CRYPTO_TEST_PSA
+	bool "Enable PSA backend and dependencies"
+	help
+	  The helper Kconfig option is used by the test to enable PSA backend
+	  and set proper TFM build profile. Used to avoid board-specific Kconfig
+	  overlays.
+
+if FP_CRYPTO_TEST_PSA
+
+choice BT_FAST_PAIR_CRYPTO_BACKEND
+	default BT_FAST_PAIR_CRYPTO_PSA
+endchoice
+
+if BUILD_WITH_TFM
+
+choice TFM_PROFILE_TYPE
+	default TFM_PROFILE_TYPE_NOT_SET
+	help
+	  The minimal TFM build profile is not configurable and cannot be used.
+endchoice
+
+endif # BUILD_WITH_TFM
```

Comment:
I would prefer not to do it. The `CONFIG_TFM_PROFILE_TYPE` is a "global" option that determines used TF-M build profile. I am not sure if enabling Fast Pair subsystem (or even fp_crypto) should modify it implicitly.

Reply:
`CONFIG_BT_FAST_PAIR_CRYPTO_PSA` depends on `!CONFIG_TFM_PROFILE_TYPE_MINIMAL`, so these options cannot be used together. Using TF-M minimal would result in errors returned by crypto APIs (`PSA_ERROR_NOT_SUPPORTED` error code is returned), because minimal TF-M profile is not configurable and does not support crypto operations (it only supports RNG).

Reply:
Oh, okay - I missed the fact that there is the dependency added. In that case, I'm fine with this approach.

Reply:
We use a different approach for Bluetooth configurations in Fast Pair where we override defaults in the FP Kconfig.

Reply:
I think it depends on what the configuration relates to. I.e. you may want to automatically tune a buffer size or turn on a feature in BLE controller, but turning on whole BLE stack under the hood may not be best approach. Similar here: the user should choose TF-M profile for the application, it should not be done automatically under the hood (but we could select set of needed features under the hood).